techiteasy.sql

```diff
@@ -0,0 +1,136 @@
+-- Database: TechItEasy
+
+-- DROP DATABASE IF EXISTS "TechItEasy";
+
+CREATE DATABASE "TechItEasy"
+    WITH
+    OWNER = postgres
+    ENCODING = 'UTF8'
+    LC_COLLATE = 'nl_NL.UTF-8'
+    LC_CTYPE = 'nl_NL.UTF-8'
+    LOCALE_PROVIDER = 'libc'
+    TABLESPACE = pg_default
+    CONNECTION LIMIT = -1
+    IS_TEMPLATE = False;
+
+COMMENT ON DATABASE "TechItEasy"
+    IS 'huiswerkopdracht bij les 8: een database voor de tv-winkel';
+	
+-- =================
+-- create tables
+-- User details table
+CREATE TABLE Person
+(
+	userID SERIAL PRIMARY KEY,
+	username VARCHAR(100) not null Unique,
+	secret VARCHAR(100) not null Unique
+);
+
+-- Product table (parent, from Java abstract class)
+CREATE TABLE Product
+(
+	productID SERIAL PRIMARY KEY,
+	productName VARCHAR(100) not null,
+	brand VARCHAR(100) not null,
+	price DECIMAL(10,2),
+	currentStock INT,
+	sold INT
+);
+
```

Comment:
Je hebt een parent tabel (product) gemaakt en per product een tabel voor de product specifieke kenmerken, dat is wel een slimme manier om alle producten in één overzichtelijke tabel te zetten